docs/Lab-EFREST/Feladat-1.md

```diff
@@ -267,3 +269,6 @@ Rétegről rétegre haladva valósítsuk meg a funkciókat.
 
 !!! example "BEADANDÓ"
     Készíts egy **képernyőképet** Postman-ből (avagy más, hasonló eszközből, ha nem Postman-t használtál), amely egy **sikertelen** beszúrást mutat. A sikertelenség oka a már létező név legyen. A képet a megoldásban `f1.png` néven add be. A képernyőképen látszódjon a **kimenő kérés és a válasz is minden részletével** (URL, body, válasz kód, válasz body). Ellenőrizd, hogy a **Neptun kódod** az URL-ben szerepel-e! A képernyőkép szükséges feltétele a pontszám megszerzésének.
+    
+!!! info "Segítség a POST kéréshez"
+    A POST kérés elküldéséhez segítséget például a hivatalos Microsoft tutorial [vonatkozó részében](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-3.1&tabs=visual-studio#examine-the-posttodoitem-create-method) találsz.
```

Comment:
@bobarna Kérdésem lenne ezzel kapcsolatban. Mi a pontos célja ennek a linknek ill. a plusz információnak? A Postman kérés segítése vagy a controller metódus megírása? A linkelt anyag túl "hosszú" (megtalálni benne, mi is az a kép vagy kódrészlet, ami a segítség tárgya), lehet, hogy célzott információ (mondjuk egy kép) célszerűbb lenne.